src/Sarif.WorkItems/SarifWorkItemsExtensions.cs

```diff
@@ -85,5 +116,28 @@ private static string ConstructFullRuleIdentifier(ReportingDescriptor reportingD
             }
             return fullRuleIdentifier;
         }
+
+        public static string CreateWorkItemDescription(this SarifLog log)
```

Comment:
Taking any and all suggestions formatting here.  I do think starting with plain text is fine to get something available.  I'll be looking at possibly just storing some very basic data and having the specific filers customize the description.

Reply:
Discussed offline with multiple parties.

---
In reply to: [394675544](https://github.com/microsoft/sarif-sdk/pull/1817#discussion_r394675544) [](ancestors = 394675544)